src/Course/ExactlyOne.hs

```diff
@@ -133,10 +133,23 @@ bindExactlyOne f (MakeExactlyOne a0) = f a0
 --   b) What is the type of `bindExactlyOne f x`?
 question_ExactlyOne_2 :: (P.String, P.String)
 question_ExactlyOne_2 =
-  ( error "todo"
-  , error "todo"
+  ( "`mapExactlyOne f x` is of type `ExactlyOne (ExactlyOne Int)` since `map` wraps the output of `f` which is already of type `ExactlyOne`."
+  , "`bindExactlyOne f x` is of type `ExactlyOne Int` since it only wraps the argument, not the result of `f`."
```

Comment:
Good. Another way to think about it, in `mapExactlyOne f`, your `a` type variable gets set to `String` and your `b` type variable gets set to `ExactlyOne Int`. Whereas, in `bindExactlyOne f`, your `a` gets set to `String` and your `b` gets set to `Int`, because `bindExactlyOne` has the `ExactlyOne _` already built in to its signature.